user/service/client/client.go

```diff
@@ -202,6 +204,23 @@ func (c *Client) Delete(ctx context.Context, id string, deleet *user.Delete, con
 			logger.WithError(err).Error("Unable to destroy profile")
 		}
 	}
+	//Sending delete user event to kafka
+	userName := *result.Username
+	logger.Infof("preparing to send delete user kafka message for %v", userName)
+	if !strings.HasSuffix(userName, "@tidepool.io") && !strings.HasSuffix(userName, "@tidepool.org") {
```

Comment:
I think this logic belongs to the consumer (i.e. the new marketo service). You might have different consumers which might be interested in all events, not only for users which don't have a tidepool account.

Reply:
copying from shoreline: "The only reason I added the tidepool email address check here (it is already in marketo) is because ginny is creating and deleting test accounts around every 3 minutes or so and for the same reason as above thought it was unnecessary logging, messages, and noise."

I can remove if does not matter

Reply:
I think it's important to capture events for all accounts even ones used only for testing. This way we can delete them in keycloak.